pimmi/download_demo.py

```diff
@@ -0,0 +1,49 @@
+import os
+from github import Github, Repository, ContentFile
+import requests
+
+
+def download_demo(dataset, dir):
+    numero_image = 0
+    if dataset == 'dataset1':
+        urls = [
+            "https://raw.githubusercontent.com/nrv/pimmi/main/demo_dataset/dataset1/000"]
+        nb_images = [1598]
+    if dataset == 'small_dataset':
+        urls = ["https://raw.githubusercontent.com/nrv/pimmi/main/demo_dataset/small_dataset/0/000",
+                "https://raw.githubusercontent.com/nrv/pimmi/main/demo_dataset/small_dataset/1/000"]
+        nb_images = [10, 1]
+
+        if not os.path.isdir(os.path.join(dir, "0")):
+            os.makedirs(os.path.join(dir, "0"))
+        if not os.path.isdir(os.path.join(dir, "1")):
+            os.makedirs(os.path.join(dir, "1"))
+
+        dirs = [os.path.join(dir, "0"), os.path.join(dir, "1")]
+    try:
+        for index, url_base in enumerate(urls):
```

Comment:
It could help to let the user know that we are doing something that takes some time. Maybe a loading bar using the tqdm module ?